src/lib/filters.ts

```diff
@@ -0,0 +1,33 @@
+import type { SeamWarning } from 'seamapi'
+
+import type { SeamCompositeError } from 'lib/index.js'
+
+export const accessCodeErrorFilter = (error: SeamCompositeError): boolean => {
+  return 'is_access_code_error' in error && error.is_access_code_error
+}
+
+export const accessCodeWarningFilter = (warning: SeamWarning): boolean => {
+  const relevantWarnings = [
+    'delay_in_removing_from_device',
+    'delay_in_setting_on_device',
+    'code_modified_external_to_seam',
+  ]
+
+  return relevantWarnings.includes(warning.warning_code)
+}
+
+export const deviceErrorFilter = (error: SeamCompositeError): boolean => {
+  return 'is_device_error' in error && error.is_device_error
+}
+
+export const deviceWarningFilter = (warning: SeamWarning): boolean => {
+  const relevantWarnings = [
+    'device_has_flaky_connection',
+    'third_party_integration_detected',
+    'salto_office_mode',
+    'salto_privacy_mode',
+    'ttlock_lock_gateway_unlocking_not_enabled',
+  ]
```

Comment:
Why was this list warning list selected? With this approach, if new warnings are released on the API it will require an update to the package.

Reply:
Yeah, sorry, this was in poor taste—forgot to remove